pkg/datapath/loader/loader.go

```diff
@@ -351,18 +351,34 @@ func (l *Loader) reloadDatapath(ctx context.Context, ep datapath.Endpoint, dirs
 	}
 
 	if ip := ep.IPv4Address(); ip.IsSet() {
+		scopedLog := ep.Logger(Subsystem).WithFields(logrus.Fields{
+			logfields.Path: objPath,
```

Comment:
Ah, sorry, I think I just understood your previous question :facepalm: 

We can drop the `Path` logfield I think. It isn't relevant to endpoint routes.